ext_emconf.php

```diff
@@ -3,18 +3,19 @@
 $EM_CONF['mfc_belogin_captcha'] = [
     'title' => 'Backend Login Captcha',
     'description' => 'Add an configurable captcha to the backend login after a give amount of failed login tries',
+    'category' => 'be',
     'version' => '4.0.0',
     'author' => 'Simon Schmidt, Christian Hellmund, Alexander Schnitzler',
     'author_email' => 'typo3@marketing-factory.de',
     'author_company' => 'Marketing Factory Consulting GmbH',
-    'category' => 'be',
     'priority' => 'bottom',
     'state' => 'stable',
     'clearCacheOnLoad' => 1,
     'constraints' => [
         'depends' => [
-            'typo3' => '9.4.0-9.5.99',
-            'recaptcha' => ''
+        'php' => '7.0.0-0.0.0',
+        'typo3' => '9.4.0-9.5.99',
+        'recaptcha' => ''
```

Comment:
I think we need an under boundery for the recaptcha extension version...

Reply:
You could do that. I only changed that it's installable with 9.5. Since it's not possible up to now, we dont use the extension.